docs/pages/Introduction/what-is-hyperbridge.mdx

```diff
@@ -0,0 +1,82 @@
+
+Additionally, it facilitates cross-chain message passing among connected chains, ISMP also serves as a synchronisation primitive across Hyperbridge's internal state machines. This enables its parachain cores to communicate with each other and delegate tasks. Making Hyperbridge to become the first multi-core parachain on Polkadot.
+
+### Full node level security guarantees
+With the ability to re-execute blocks of its connected chains, Hyperbridge achieves the elusive full node security in cross-chain bridges. This is accomplished by compressing the state witness, transaction data, and contract code, which can then be uncompressed and re-executed within a parachain core. This is how smart-contract parachains currently function on the Polkadot relay chain.
+
+
+## Features 
+Hyperbridge presents many features that make it a unique and powerful tool for developers. Here are some of the key features and tools that Hyperbridge offers:
+
+### General Message passing 
+Hyperbridge allows for general message passing among connected chains. This allows developers to send arbitrary data to connected chains, as well as read the storage of applications on connected chains. This is done through the ISMP API, which provides a familiar HTTP-like interface for developers to interact with.
+
+### Gateway
+Gateway is the dApp component of Hyperbridge. It allows developers bridge tokens across chains, and also provides a user-friendly interface for users to interact with the bridge. Gateway is built on top of the ISMP API, and provides a simple and intuitive way for users to bridge tokens across chains. You can play around with the Gateway dApp on the Hyperbridge testnet (here)[https://gateway.hyperbridge.network/].
+
+## Use Cases
+Hyperbridge's unique architecture and features enable a wide range of use cases across various industries and applications. Here are some potential use cases where developers can leverage Hyperbridge to create solutions:
+
+1. **Cross-Chain Asset Transfers**: Hyperbridge facilitates seamless transfer of assets across different blockchains. This capability opens up various use cases, such as:
+   - **Asset Diversification**: Traders and investors can easily diversify their portfolios across multiple chains without facing the traditional barriers associated with transferring assets between incompatible networks.
+   - **Cross-Border Payments**: Hyperbridge can be leveraged to create efficient cross-border payment solutions, allowing for cost-effective transfers of value between individuals and businesses operating in different jurisdictions.
```

Comment:
Most of these use cases are not really in line with hyperbridge.
Here are some:
Cross-Chain Asset Transfers
Cross Chain Governance
Trustless Oracles for onchain data
Multichain Liquid Staking and Restaking